iTechArt.TicTacToe.Foundation/FigureFactories/CircleFactory.cs

```diff
@@ -0,0 +1,14 @@
+﻿using iTechArt.TicTacToe.Foundation.Figures;
+using iTechArt.TicTacToe.Foundation.Interfaces;
+
+namespace iTechArt.TicTacToe.Foundation.FigureFactories
+{
+    class CircleFactory : IFigureFactory
+    {
+        public IFigure CreateFigure()
```

Comment:
Merge these factories into single `FigureFactory` with parameter figure type.
Move it to the figures folder.
Remove FigureFactories folder.